src/sentry/options/defaults.py

```diff
@@ -552,6 +552,11 @@
 register("github-app.private-key", default="", flags=FLAG_CREDENTIAL)
 register("github-app.client-id", flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE)
 register("github-app.client-secret", flags=FLAG_CREDENTIAL | FLAG_PRIORITIZE_DISK)
+register(
+    "github-app.get-trees-refactored-code",
```

Comment:
I will be turning this into a feature rather than an option.

Reply:
Are you still planning to make this change? I'm curious if there's a use case we're missing from flagpole that makes this preferable.

Reply:
Switching over won't be necessary after all. Thanks for checking!